docs/installationguide.md

```diff
@@ -98,7 +98,9 @@ There are also some global configuration options:
 -   **configRetrieval**: this flag indicates whether the incoming notifications to the IoTAgent should be processed
     using the bidirectionality plugin from the latest versions of the library or the JSON-specific configuration
     retrieval mechanism (described in the User Manual). Simultaneous use of both mechanisms is not allowed.
--   **compressTimestamp**: this flags enables the timestamp compression mechanism, described in the User Manual.
+-   **compressTimestamp**: this flag enables the timestamp compression mechanism, described in the User Manual.
+-   **multiCore**: this (optional) flag enables the execution of the IoT Agent in multi-cores if it is `true` or in
```

Comment:
Not fully sure but I think to remember that the criteria is that library variables are described in library installation guide documentation. In agents repositories, only variables introduced by the agent (i..e mqtt and http things) are described.

@AlvaroVega do you remember how we are dealing with this in docs?

Reply:
Do you mead this: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/installationguide.md#configuration-using-environment-variables ?

Reply:
Yes.

My point is that if a given config parameter / env var is described in iotagent-node-lib documentation (because its processing belong to the library) that config parameters / env var is not described in agent's documentation. Is that correct?

Reply:
I have checked this with @AlvaroVega . It is confirmed my point, so please remove the multiCore setting from this documentation.

Reply:
Fixed in 7a2bf08